frontend/plugins/insurance_ui/src/modules/insurance/components/vendors/VendorsColumns.tsx

```
@@ -28,9 +29,10 @@ export const vendorsColumns: ColumnDef<InsuranceVendor>[] = [
   {
     id: 'name',
     accessorKey: 'name',
-    header: () => (
-      <RecordTable.InlineHead icon={IconBuilding} label="Vendor Name" />
-    ),
+    header: () => {
+      const { t } = useTranslation('insurance');
```

ESLint

[error] React Hook "useTranslation" is called in function "header" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)